css-text-4/Overview.bs

```diff
@@ -4603,8 +4456,35 @@ Breaking Rules for Letters: the 'word-break' property</h3>
 			(which uses [=spaces=] between words),
 			and is also useful for mixed-script text where CJK snippets are mixed
 			into another language that uses [=spaces=] for separation.
+
+		<dt><dfn>auto</dfn>
+		<dd>
+			This value directs the user agent to perform language-specific content analysis
+			to determine where to insert [=virtual word boundaries=].
+
+            User agents may activate
+            language-specific content analysis
+            in response to user preferences.
+            User agents with this behavior must do this
+            by setting the [=declared value=] of 'word-break' to ''word-break/auto''
+            in the [=User Origin=].
```

Comment:
Probably due to my English skill, sorry, but I failed to understand what this sentence emeans. Could you consider rephrasing?

Reply:
Oh, this just means "if a browser turns this on by default, they should make `getComputedStyle()` reflect the fact that the feature is enabled"

Reply:
I'll add a note which should hopefully make this more clear.